evamaina/JavaDevsNairobi
# #155820624 Launch the DetailActivity from the MainActivity

#### What does this PR do?
- Launch the DetailActivity from the MainActivity
#### Description of Task to be completed?
- Ensure that when a user clicks on an item in the list that a DetailActivity is launched.
- Implement back button functionality
#### How should this be manually tested?
- Clone the project, git clone https://github.com/evamaina/JavaDevsNairobi.git.
- On the terminal, cd into JavaDevsNairobi directory.
- Checkout to branch `ft-launch-detailActivity-15582062`
- Run the app and on the emulator, when the main activity launches, click on the view detail card to launch the DetailActivity.
- You should be able to see a screen resembling the one shown on the screenshots below.
#### Any background information you want to provide
I launched the detailActivity to the mainActivity when creating a layout file for the detail activity.In this pull request I have only implemented the functionality for the back button in the detail activity so that when a user clicks on an item and that activity is launched, he she can be able to navigate back to the mainActivity page.

#### What are the relevant pivotal tracker stories?
[#155820624](https://www.pivotaltracker.com/story/show/155820624)
#### Screenshots
![image](https://res.cloudinary.com/dqlop17s0/image/upload/v1548233477/shot.gif)
